<launch>
  <arg name="config" default="$(find-pkg-share autoware_command_mode_decider)/config/default.param.yaml"/>
  <node pkg="autoware_command_mode_decider" exec="command_mode_decider_node">
    <param from="$(var config)"/>
    <remap from="~/command_mode/availability" to="/system/command_mode/availability"/>
    <remap from="~/command_mode/status" to="/system/command_mode/status"/>
    <remap from="~/command_mode/request" to="/system/command_mode/request"/>
    <remap from="~/operation_mode/state" to="/system/operation_mode/state"/>
    <remap from="~/operation_mode/change_operation_mode" to="/system/operation_mode/change_operation_mode"/>
  </node>
  <node pkg="autoware_command_mode_decider" exec="availability_converter_node">
    <remap from="~/command_mode/availability" to="/system/command_mode/availability"/>
    <remap from="~/operation_mode/availability" to="/system/operation_mode/availability"/>
  </node>
</launch>
